src/Output.php

```diff
@@ -85,6 +85,78 @@ public function writeResult(Result $result, ErrorFormatter $errorFormatter, $ign
     }
 }
 
+class GitLabOutput implements Output
```

Comment:
It might be a good idea to merge this after the PSR-4 changes have been pushed through. Otherwise, the `src/Output.php` file keeps growing. See #45.

Reply:
PR from the current state is ok. Is my fault that #45 is not done.